R/applications.R

```diff
@@ -592,6 +589,38 @@ application_nasnet <- function(input_shape = NULL, penultimate_filters = 4032L,
   
 }
 
+#' @rdname application_nasnet
+#' @export
+application_nasnetlarge <- function(input_shape = NULL, include_top = TRUE, weights = NULL, 
+                               input_tensor = NULL, pooling = NULL, classes = 1000) {
+  
+  keras$applications$NASNetLarge(
```

Comment:
You need all of the same `as.integer` and `as_nullable_integer` calls here as in the other applications.